.circleci/config.yml

```diff
@@ -1,54 +1,108 @@
 version: 2.1
+
 orbs:
   slack: circleci/slack@3.4.2
 
-jobs:
-  build:
+executors:
+  docker-executor:
     docker:
       - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester-v4
```

Comment:
Can this get pushed to the most recent `stitch-tap-tester` rather than `tap-tester-v4`